versioned_docs/version-0.4.2/tutorials/react-native.mdx

```diff
@@ -587,11 +622,11 @@ const styles = StyleSheet.create({
 export default RoomScreen;
 ```
 
-You should now see your own camera on your mobile device.  
-You can add another participant and their new track (displaying for example rotating frog) in the dashboard like
-this:
+You should now see your own camera on your mobile device.
+You can add another participant and their new track (displaying for example rotating heart) in the dashboard like
+this (for this version of mobile sdk 0.1.6 remember check attach metadata checkbox) :
```

Comment:
For version 0.1.6 it is necessary for ios otherwise remote videos will not be displayed. This is a bug because ios did not properly deserialize metadata if it was null. This is fixed on master but master is not yet released